shipstation/api.py

```diff
@@ -398,3 +418,23 @@ def post(self, endpoint='', data=None):
         )
         if self.debug:
             pprint.PrettyPrinter(indent=4).pprint(r.json())
+
+    def fetch_orders(self, parameters_dict=None):
```

Comment:
In an effort to move documentation on this project to Sphinx I'm going to begin enforcing docstrings for methods, in the [Google style](http://www.sphinx-doc.org/es/stable/ext/example_google.html).

Can you add a block here detailing the function and its usage? e.g.,
```python
def fetch_orders(self, parameters_dict=None):
    """
    Query and fetch existing orders from ShipStation

    Args:
        parameters_dict (dict): [short explaination]

    Raises:
        AttributeError: [explain why]

    Examples:
        >>> ss.fetch_orders(parameters_dict={'order_status': 'shipped', 'page': '2'})
```

Reply:
Additionally, I don't think we need to explain the type in argument names. I'd prefer `query_parameters` or just `parameters`